media/css/firefox/welcome.scss

```diff
@@ -176,3 +200,76 @@
     background-color: $color-off-black;
     color: $color-white;
 }
+
+
+//* -------------------------------------------------------------------------- */
+//  Modal
+
+.modal-logo {
+    @include at2x('/media/protocol/img/logos/firefox/browser/logo-lg.png', auto, 100%);
+    @include image-replaced;
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+.mzp-c-modal-window {
+    padding: $spacing-sm;
+}
+.mzp-c-modal-inner {
+    background-color: transparent;
+    padding: 0;
+
+    h3 {
+        @include text-display-md;
+        color: $color-off-black;
+        margin-bottom: 0;
+
+        + p {
+            margin-bottom: $layout-md;
+        }
+    }
+
+    .mzp-c-modal-overlay-contents {
+        background: $color-white;
```

Comment:
The white background here is making the small links inside the modal invisible, so you may need to add some link styles here to compensate.